Remove AWS from unsupported default-domain in Caveat (#2574)
knative/docs: docs/install/any-kubernetes-cluster.md

kubectl apply --filename {{< artifact repo="serving" file="serving-default-domain.yaml" >}}
```

> **Caveat**: This will only work if the cluster LoadBalancer service exposes an IPv4 address or hostname, so it will not work with IPv6 clusters or local setups like Minikube.  For these, see "Real DNS" or "Temporary DNS".

Reviewer: ```suggestion
**Caveat**: This will only work if the cluster LoadBalancer service exposes an IPv4 address or hostname. It does not work for IPv6 clusters or local setups, such as Minikube.
```

> **Caveat**: This will only work if the cluster LoadBalancer service exposes an IPv4 address or hostname, so it will not work with IPv6 clusters or local setups like Minikube.  For these, see "Real DNS" or "Temporary DNS".

Reviewer: Add a line after this:
For information about IPv6 clusters or local setups, see [then add links to docs about DNS types instead of saying "Real DNS" or "Temporary DNS"].

Reviewer 2: The "Real DNS" and "Temporary DNS" are other tabs (see lines 300-363).
